node/trace/agent.js

```diff
@@ -122,7 +136,17 @@ Agent.prototype.setupNewSpan = function setupNewSpan(options) {
         flags: options.flags
     });
 
-    var parentSpan = self.getCurrentSpan();
+    var parentSpan;
+    if (self.asyncListener) {
+        parentSpan = self.getCurrentSpan();
+    } else {
+        parentSpan = options.parentSpan;
+        if (options.outgoing) {
+            self.logger.error("TChannel tracer: parent span not specified " +
```

Comment:
there's no need for these line length gymnastics in this repo, we don't use a line-length limiting linter...

also this seems like a warn to me not an error